pnpm-lock.yaml

```diff
@@ -50,6 +49,23 @@ importers:
       '@sveltejs/kit': link:../../packages/kit
       svelte: 3.40.0
 
+  examples/i18n:
+    specifiers:
+      '@fontsource/fira-mono': ^4.2.2
+      '@lukeed/uuid': ^2.0.0
+      '@sveltejs/adapter-static': ^1.0.0-next.13
+      '@sveltejs/kit': workspace:*
+      cookie: ^0.4.1
+      svelte: ^3.34.0
+    dependencies:
+      '@fontsource/fira-mono': 4.2.2
+      '@lukeed/uuid': 2.0.0
+      cookie: 0.4.1
+    devDependencies:
+      '@sveltejs/adapter-static': link:../../packages/adapter-static
+      '@sveltejs/kit': link:../../packages/kit
+      svelte: 3.38.2
```

Comment:
The CI is complaining that you have a broken lockfile

Reply:
was part of the test project and is removed